extensions/ql-vscode/package.json

```diff
@@ -533,6 +534,10 @@
         "command": "codeQLQueryHistory.showEvalLogSummary",
         "title": "Show Evaluator Log (Summary)"
       },
+      {
+        "command": "codeQLQueryHistory.showEvalLogVisualizer",
+        "title": "Show Evaluator Log (Visualizer)"
```

Comment:
Just as a note, the summary file we are exposing to users (at least up until now) isn't the summary JSON file — it's the "human-readable" text version of the summary file. Our assumption was that users would prefer to have a easily consumable version of the summary. 

I _could_ change it to display the summary JSON file (which is used to build the UI) — this would align better with a future "jump to log" command in the UI. Right now, the log it would "jump to" would be neither one of the logs shown with other commands, which would be really confusing.

On the other hand, we could defer that change until the "jump to log" command is actually implemented. In that case maybe the commands could be `Raw JSON`, `Summary Text`, and `UI`?

Reply:
Ah! I hadn't realised the summary was text. You are right of course. I like the distinction of `Raw JSON`, `Summary Text`, and `UI`.

Reply:
Done 😄